Cherry-pick translations from android14 weblate

lucasmz-dev opened the pull request:
#668 but for `android15`

Weblate is locked so this should be safe (https://hosted.weblate.org/projects/calyxos/seedvault/#information)

CC: @chirayudesai @mikeNG

chirayudesai commented:
Another option: Directly edit target branch of https://github.com/seedvault-app/seedvault/pull/668 - but I don't know if that'll mess anything up upstream.

Not merging it to 14 directly since the translations will end up there anyways via the backports.

lucasmz-dev commented:
@chirayudesai doesn't work well:
https://github.com/lucasmz-dev/CONTRIB-seedvault/pull/1

merges extra code reverts and changes and can't be done without dealing with conflicts

weblate committed:
Import translations from Weblate

Co-authored-by: Andi Chandler <andi@gowling.com>
Co-authored-by: Anonymous <noreply@weblate.org>
Co-authored-by: Fjuro <fjuro@alius.cz>
Co-authored-by: Hasáky gum <caro.jf@protonmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Idcrafter <minefater1@gmail.com>
Co-authored-by: Igor Sorocean <sorocean.igor@gmail.com>
Co-authored-by: Jerguš Fonfer <caro.jf@protonmail.com>
Co-authored-by: Kristian Nordin <kristian.po@proton.me>
Co-authored-by: Languages add-on <noreply-addon-languages@weblate.org>
Co-authored-by: LucasMZ <git@lucasmz.dev>
Co-authored-by: Michael Bestas <mkbestas@gmail.com>
Co-authored-by: Milo Ivir <mail@milotype.de>
Co-authored-by: Oğuz Ersen <oguz@ersen.moe>
Co-authored-by: Renko <kull@tutanota.de>
Co-authored-by: Ricky Tigg <ricky.tigg@gmail.com>
Co-authored-by: Serhii Horichenko <serhii@horichenko.com>
Co-authored-by: Yaron Shahrabani <sh.yaron@gmail.com>
Co-authored-by: abc0922001 <abc0922001@hotmail.com>
Co-authored-by: gallegonovato <fran-carro@hotmail.es>
Co-authored-by: hhs bbsb <bamboo@ik.me>
Co-authored-by: hsfg6 <hsfg6@proton.me>
Co-authored-by: ikya91 <weblate.haq9g@simplelogin.com>
Co-authored-by: lucasmz-dev <git@lucasmz.dev>
Co-authored-by: nautilusx <translate@disroot.org>
Co-authored-by: nobody <info@localcdn.org>
Co-authored-by: pa cs <cs.pa@users.noreply.hosted.weblate.org>
Co-authored-by: rehork <cooky@e.email>
Co-authored-by: sysxda <coderz35@gmail.com>
Co-authored-by: Максим Горпиніч <mgorpinic2005@gmail.com>
Co-authored-by: தமிழ்நேரம் <anishprabu.t@gmail.com>
Co-authored-by: さしみ <kazasiuun@gmail.com>
Co-authored-by: 大王叫我来巡山 <hamburger2048@users.noreply.hosted.weblate.org>
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/ca/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/cs/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/de/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/el/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/en_GB/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/es/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/fi/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/fr/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/he/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/hr/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/ja/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/nl/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/pl/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/ro/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/ru/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/sk/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/sv/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/ta/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/tr/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/uk/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_contactsbackup/en_GB/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_contactsbackup/hr/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_contactsbackup/sk/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_contactsbackup/ta/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/cs/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/de/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/el/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/en_GB/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/es/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/he/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/ja/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/pl/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/ro/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/sk/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/sv/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/ta/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/tr/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/uk/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/zh_Hans/
Translation: CalyxOS/Seedvault
Translation: CalyxOS/Seedvault contacts backup
Translation: CalyxOS/Seedvault storage backup
